tests/integration/targets/mso_tenant/tasks/main.yml

```diff
@@ -309,7 +316,7 @@
 - name: Verify nm_add_tenant3 (when mso_username == admin)
   ansible.builtin.assert:
     that:
-    - nm_add_tenant3.current.userAssociations | length == 1
+    - nm_add_tenant3.current.userAssociations | length == 2
```

Comment:
The `userAssociations` length is 2, because the current inventory `mso_username` is `admin` and `r_ansible_github_ci` remote user is associated with same tenant.

Reply:
mso_username in the PR and in the repo is mso_username=ansible_github_ci and not admin so this is probably wrong.

Reply:
Done